src/main/java/com/google/cloud/tools/gradle/appengine/core/AppEngineCorePlugin.java

```diff
@@ -94,6 +109,96 @@ public void execute(Project project) {
     });
   }
 
+  private void createDeployCronTask() {
+    project.getTasks().create(DEPLOY_CRON_TASK_NAME, DeployCronTask.class, new Action<DeployCronTask>() {
+      @Override
+      public void execute(final DeployCronTask deployTask) {
+        deployTask.setGroup(APP_ENGINE_TASK_GROUP);
+        deployTask.setDescription("Deploy Cron configuration");
+
+        project.afterEvaluate(new Action<Project>() {
```

Comment:
I don't think it can infer type in this situation.
The signature uses a wildcard for that parameter. `<? extends Project>`